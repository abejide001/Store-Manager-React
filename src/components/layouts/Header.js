import React from 'react';
import { Link } from 'react-router-dom';
import '../../assets/css/style.css';
import logo from '../../assets/images/logo.png';

export default function Header() {
  return (
    <div>
      <header>
        <div className="container">
          <div id="header-title">
            <img src={logo} alt="logo" />
          </div>
          <nav>
            <ul>
              <li>
                <Link to="/login">Login</Link>
              </li>
            </ul>
          </nav>
        </div>
      </header>
    </div>
  );
}
