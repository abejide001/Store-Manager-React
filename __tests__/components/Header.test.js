/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../../src/components/layouts/Header';

describe('<Header />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<Header />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
