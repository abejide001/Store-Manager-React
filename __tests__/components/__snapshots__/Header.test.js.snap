// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> render() renders the component 1`] = `
<div>
  <header>
    <div
      className="container"
    >
      <div
        id="header-title"
      >
        <img
          alt="logo"
          src="test-file-stub"
        />
      </div>
      <nav>
        <ul>
          <li>
            <Link
              replace={false}
              to="/login"
            >
              Login
            </Link>
          </li>
        </ul>
      </nav>
    </div>
  </header>
</div>
`;
